internal/cmd/cluster/command_describe.go

```diff
@@ -56,6 +56,8 @@ func newDescribeCommand(prerunner pcmd.PreRunner, userAgent string) *cobra.Comma
 	cmd.Flags().String("ca-cert-path", "", "Self-signed certificate chain in PEM format.")
 	pcmd.AddOutputFlag(cmd)
 
+	cobra.CheckErr(cmd.MarkFlagFilename("ca-cert-path"))
```

Comment:
Does this match anything regardless of extension if none are given here?

Reply:
When no extension is given, it doesn't do anything as far as I can tell. I'll go through and delete these.